### PR TITLE
sys-devel/crossdev: Move to EAPI 6 and git-r3

### PR DESCRIPTION
**Pull request by bjacquin**
Package-Manager: portage-2.2.28

### CODE REVIEW

**Comment by floppym** [comment]
cc @vapier

**Comment by gktrk** [comment]
potentially @lu-zero and @blueness

**Comment by monsieurp** [comment]
ping

File: sys-devel/crossdev/crossdev-20160602-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+inherit eutils
```
Comment (mgorny) [inline sys-devel/crossdev/crossdev-20160602-r1.ebuild]: eutils doesn't seem to be used anymore.

File: sys-devel/crossdev/crossdev-20160602-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+DEPEND="app-arch/xz-utils"
+
+PATCHES=(
+	"${FILESDIR}"/use-new-path-for-functions.sh.patch
+)
+
+src_install() {
+	default
+	if [[ "${PV}" == "99999999" ]] ; then
+		sed -i "s:@CDEVPV@:${EGIT_VERSION}:" "${ED}"/usr/bin/crossdev || die
```
Comment (mgorny) [inline sys-devel/crossdev/crossdev-20160602-r1.ebuild]: `${ED}` ends with a slash in EAPI 6 and older. Please use `${ED%/}` to account for that portably.

**Comment by blueness** [comment]
It looks sane to me but its @vapier 's call.

**Pull request closed without merging**